src/client/theme-default/components/VPSidebarLink.vue

```diff
@@ -34,55 +30,43 @@ watchEffect(() => {
 
 <template>
   <VPLink
-    class="link"
+    class="VPSidebarLink"
     :class="{ active }"
-    :style="{ paddingLeft: 16 * (depth - 1) + 'px' }"
     :href="item.link"
     :tabindex="isSidebarEnabled || isSidebarOpen ? 0 : -1"
-    @click="closeSideBar"
     ref="link"
+    @click="closeSideBar"
   >
-    <span v-html="item.text" class="link-text" :class="{ light: depth > 1 }"></span>
+    <span class="link-text" v-html="item.text" />
   </VPLink>
-  <template
-    v-if="'items' in item && depth < maxDepth"
-    v-for="child in item.items"
-    :key="child.link"
-  >
-    <VPSidebarLink :item="child" :depth="depth + 1" />
-  </template>
```

Comment:
I have also noticed an issue that in this PR would limit the maximum navlist depth allowed to two levels, whereas before it was unlimited, so this PR will cause a breaking change. Therefore, sites used more than two levels of items in the navigation list previously, will not be able to upgraded to a new VitePress version due to breaking change, they can only choose to create a new theme by themselves or reorganize the navigation list of the project

Reply:
It wasn't documented, and VitePress is in alpha. Yes, this is breaking change.